### ISSUE TITLE
[FEATURE] Add some more Sensors for LWZ

### ISSUE THREAD

**Issue by northalpha** [pail23/stiebel_eltron_isg_component#51]
I really like you integration and it might be replacing my clumpsy modbus config ;)

May i ask you if you may add following sensor to the integration?

```
      - name: "LWZ VORLAUFTEMP"
        slave: 1
        address: 11
        input_type: input
        unit_of_measurement: °C
        device_class: temperature
        scale: 0.1
        precision: 1

      - name: "LWZ RUECKLAUFTEMP"
        slave: 1
        address: 12
        input_type: input
        unit_of_measurement: °C
        device_class: temperature
        scale: 0.1
        precision: 1

      # Power consumption
      - name: "LWZ P-HEIZUNG-TAG"
        slave: 1
        address: 3021
        input_type: input
        unit_of_measurement: kWh
        device_class: energy
      - name: "LWZ P-HEIZUNG-SUMME MWh"
        slave: 1
        address: 3023
        input_type: input
        unit_of_measurement: MWh
        device_class: energy
      - name: "LWZ P-HEIZUNG-SUMME kWh"
        slave: 1
        address: 3022
        input_type: input
        unit_of_measurement: kWh
        device_class: energy
      - name: "LWZ P-WW-TAG"
        slave: 1
        address: 3024
        input_type: input
        unit_of_measurement: kWh
        device_class: energy
      - name: "LWZ P-WW-SUMME MWh"
        slave: 1
        address: 3026
        input_type: input
        unit_of_measurement: MWh
        device_class: energy
      - name: "LWZ P-WW-SUMME kWh"
        slave: 1
        address: 3025
        input_type: input
        unit_of_measurement: kWh
        device_class: energy

      # Running times
      - name: "LWZ VERDICHTER-HEIZEN"
        slave: 1
        address: 3027
        input_type: input
        unit_of_measurement: h
      - name: "LWZ VERDICHTER-WW"
        slave: 1
        address: 3029
        input_type: input
        unit_of_measurement: h
      - name: "LWZ ELEKTR-NE-HEIZEN"
        slave: 1
        address: 3030
        input_type: input
        unit_of_measurement: h
      - name: "LWZ ELEKTR-NE-WW"
        slave: 1
        address: 3031
        input_type: input
        unit_of_measurement: h

```

Also i am trying to get the total of `VERDICHTERSTARTS` from modbus via

```
      - name: "LWZ VERDICHTERSTARTS"
        slave: 1
        address: 31
        input_type: input
```
but i do not see these are realistic values:

![Bildschirmfoto 2023-05-05 um 14 24 46](https://user-images.githubusercontent.com/6721009/236456979-22858402-fa66-4c7c-b6de-f8e9b015c933.png)

vs.

![Bildschirmfoto 2023-05-05 um 14 23 38](https://user-images.githubusercontent.com/6721009/236456982-eb16b2f7-06d7-41ee-9a0e-313fe62e6725.png)

maybe you have more insights? I am using an LWZ 5S plus here.

**Comment by pail23** [comment]
The missing entities should be implemented in 2023.05. I appreciate testing since I don't have a LWZ/LWA model. 

For the Verdichterstarts: I think it's register 30 (not 31) according the documentation. Please check if my implementation works.

**Comment by northalpha** [comment]
Yes they aree there, thanks, i might would choose another name, but thats okay for me.

Unfortunately the `VERDICHTERSTARTS` seems to give us a valid, non changing "number" but this is not correct either:

ISG:
![Bildschirmfoto 2023-05-09 um 13 11 22](https://github.com/pail23/stiebel_eltron_isg_component/assets/6721009/0a2c4e44-4e2c-421c-8cb9-b364c1ad5a51)


HA:
![Bildschirmfoto 2023-05-09 um 13 12 09](https://github.com/pail23/stiebel_eltron_isg_component/assets/6721009/03149a23-d4e5-4f53-b3c7-35fac4fa1c1b)

**Comment by pail23** [comment]
I guess I have found a issue: The register for the compressor start was wrong. Could you please test again with version 2023.5.1? In case it is not working: You can now download diagnostics information an attach them to this issue.

**Comment by northalpha** [comment]
Integration reads: `6`
ISG shows: `6676`

Maybe this is something strange with lower / higher bit/int in the registers? Or maybe my ISG Web Version `12.1.2` having some issues. i have contacted Stiebel Support and asked them if there is any hint or bugs known for this.

thanks for your support and patience.

**Comment by pail23** [comment]
Currently I use the modbus register 31 for the compressor starts sensor. Looking at your data I have the following hypothesis:

Compressor starts = register_31 *1000  + register 34 (This would be 6677 in the attached Diag Data). Maybe you could verify this by another Diag Data Dump.

**Comment by northalpha** [comment]
Yes! Epic discovery!

ISG Reads: 6695

Dump:

```
        "compressor_starts": 6,
        "system_values": [
          250,
          250,
          0,
          250,
          250,
          0,
          112,
          265,
          295,
          265,
          325,
          252,
          265,
          15,
          0,
          536,
          460,
          29,
          115,
          25,
          115,
          25,
          253,
          36,
          32768,
          32768,
          32768,
          313,
          1622,
          1184,
          6,
          0,
          0,
          695, <--- here it is ... :)
          32768,
          32768,
          32768,
          32768,
          32768,
          32768
        ],
```
so the formula you suggest adds up! 🎉

**Comment by thorsten** [comment]
Some ventilation values are missing as well:

      - name: "LWZ ZULUFT-IST-LUEFTERDREHZAHL"
        slave: 1
        address: 17
        input_type: input
        unit_of_measurement: Hz
      - name: "LWZ ZULUFT-SOLL-VOLUMENSTROM"
        slave: 1
        address: 18
        input_type: input
        unit_of_measurement: m³/h
      - name: "LWZ ABLUFT-IST-LUEFTERDREHZAHL"
        slave: 1
        address: 19
        input_type: input
        unit_of_measurement: Hz
      - name: "LWZ ABLUFT-SOLL-VOLUMENSTROM"
        slave: 1
        address: 20
        input_type: input
        unit_of_measurement: m³/h
      - name: "LWZ STUFE-TAG"
        slave: 1
        address: 1017
        input_type: holding
      - name: "LWZ STUFE-NACHT"
        slave: 1
        address: 1018
        input_type: holding

**Comment by thorsten** [comment]
@pail23 I have a Stiebel Eltron LWZ 304 Trend and I can do some testing.

**Comment by northalpha** [comment]
LWZ 5s plus here for testing ready :)

**Comment by pail23** [comment]
@thorsten I have implemented the registers in 2023.5.4. 1017 was already there. It's now called fan level day

**Comment by pail23** [comment]
Yes, this was not yet fixed in the last release. It's fixed in the main branch an will be released with the next version.

**Comment by pail23** [comment]
@northalpha This should now be implemented in 2023.6

**Comment by workedbeforepush** [comment]
Hi @pail23 I have a Tecalor TTL 5.6 ACS heat pump and the integration is working pretty good so far. I was wondering why the compressor starts are only emitted for LWZ models. In the ISG Servicewelt i can see the "Verdichterstarts" value so i was assuming that it will also work for my 5.6 ACS.
